apps/mark/frontend/src/pages/admin_screen.tsx

```diff
@@ -54,11 +56,22 @@ export function AdminScreen({
   logger,
 }: AdminScreenProps): JSX.Element {
   const election = electionDefinition?.election;
+  const electionHash = electionDefinition?.electionHash;
 
-  const [isFetchingElection, setIsFetchingElection] = useState(false);
-  function loadElection() {
-    setIsFetchingElection(true);
-    fetchElection();
+  const electionDefinitionFromCardQuery =
+    getElectionDefinitionFromCard.useQuery(electionHash, {
+      // Disable automatic fetching and only allow manual fetching through .refetch()
+      enabled: false,
+    });
+
+  async function loadElection() {
+    const { data } = await electionDefinitionFromCardQuery.refetch();
+    assert(data !== undefined);
+    const electionDefinitionFromCard = data.ok();
+    // TODO: Handle case that electionDefinitionFromCard is undefined, e.g. because it's malformed
```

Comment:
This isn't a regression - we already weren't handling this case. I'm just taking note